src/query.rs

```diff
@@ -318,13 +318,13 @@ macro_rules! add_lints {
             )*
         }
 
-        fn get_query_text_contents() -> Vec<String> {
+        // No way to avoid this lint -- the push() calls are macro-generated.
+        #[allow(clippy::vec_init_then_push)]
+        fn get_query_text_contents() -> Vec<&'static str> {
             let mut temp_vec = Vec::new();
             $(
                 temp_vec.push(
-                    std::fs::read_to_string(
-                        format!("./src/lints/{}.ron", stringify!($name))
-                    ).unwrap()
+                    include_str!(concat!("lints/", stringify!($name), ".ron"))
```

Comment:
Why concat and not the format that I wrote?

Reply:
`include_str!()` demands a string *literal*, not just a string-like value:
![image](https://user-images.githubusercontent.com/2348618/208532112-887d9f15-9a7c-46de-9f72-472397ba71b4.png)